app/policies/application_policy.rb

```diff
@@ -53,11 +53,17 @@ def resolve
 
   private
 
-  def user_is_owner_or_admin?
-    record.user == user || user.admin? if !user.nil? || record.organization.organization_owner?(user)
+  def user_is_super_team_member?
```

Comment:
I have a question, do we need something like this for the user as admin as well? Check both if they're admin and a member of the team?

Reply:
from what I think, admin will be us (god user) and then we have super_user which will be each super_team_member (organization admin? lmk if that's what you were talking about